HealthcareSystem/PatientWebApp/Controllers/FeedbackController.cs

```diff
@@ -48,6 +52,8 @@ public IActionResult GetFeedbackById(int id)
         /// </summary>
         /// <param name="feedbackDTO">an object to be added to the database</param>
         /// <returns>if alright returns code 200(Ok), if not 400(bed request)</returns>
+        /// 
+        [Authorize(Roles = UserRoles.Patient)]
         [HttpPost]
         public ActionResult AddFeedback(FeedbackDTO feedbackDTO)
         {
```

Comment:
You should probably also check here that the jmbg in the DTO matches the principal.

Reply:
Fixed.